plugins/performance-lab/includes/site-health/audit-enqueued-assets/helper.php

```diff
@@ -287,7 +287,7 @@ function perflab_aea_get_total_size_bytes_enqueued_styles() {
  * @return string Returns absolute path to the resource.
  */
 function perflab_aea_get_path_from_resource_url( string $resource_url ): string {
-	if ( ! $resource_url ) {
+	if ( '' === $resource_url ) {
 		return '';
```

Comment:
Could just return the value early in this case.
```suggestion
	if ( '' === $resource_url ) {
		return $resource_url;
```